src/Rings/mpoly-ideals.jl

```diff
@@ -714,10 +725,9 @@ Computes the radical.
   return is_subset(radical(I), I)
```

Comment:
In the invisible line above, shouldn't `has_attribute(I, :is_prime) && is_prime(I) && return true` be turned into `is_known(is_prime, I) && is_prime(I) && return true` ?